lime/lime_tabular.py

```diff
@@ -316,9 +316,19 @@ def explain_instance(self,
         feature_names = copy.deepcopy(self.feature_names)
         if feature_names is None:
             feature_names = [str(x) for x in range(data_row.shape[0])]
-
-        values = self.convert_and_round(data_row)
-
+        # get column names if numpy.ndarray or pandas df:
+        try:
```

Comment:
Could we possibly check if the type is a pandas df rather than doing that via try / catch? It's a little awkward to have the most common use case be in an exception.